ha/test/integration/event_analyzer/test_alert_filter.py

```diff
@@ -27,14 +27,14 @@
     from cortx.utils.conf_store.conf_store import Conf
```

Comment:
Instead of changing it in alert_filter.py, can you create a new file for ClusterResourceFilter and ClusterResourceParser separately?